pkg/apis/capability/v1/validation.go

```diff
@@ -15,3 +15,13 @@ func (req *InstallRequest) Validate() error {
 	}
 	return nil
 }
+
+func (req *SyncRequest) Validate() error {
+	if req.Cluster != nil && req.Cluster.Id != "" {
+		// empty string indicates "all" here
```

Comment:
should add this comment to the SyncRequest message definition, for IDE completion